samples/xamarin-forms/Cinephile/Cinephile.ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reactive;
+using System.Reactive.Concurrency;
+using ReactiveUI;
+using Splat;
+
+namespace Cinephile.ViewModels
+{
+    public abstract class ViewModelBase : ReactiveObject, IRoutableViewModel, ISupportsActivation
+    {
+        public string UrlPathSegment
+        {
+            get;
+        }
+
+        public IScreen HostScreen
+        {
+            get;
+        }
+
+        protected readonly ViewModelActivator viewModelActivator = new ViewModelActivator();
+        public ViewModelActivator Activator => viewModelActivator;
+
+        protected readonly Interaction<AlertViewModel, Unit> _showAlert;
+        public Interaction<AlertViewModel, Unit> ShowAlert => _showAlert;
+
+        protected readonly Interaction<string, Unit> _openBrowser;
+        public Interaction<string, Unit> OpenBrowser => _openBrowser;
+
+
+        protected readonly IScheduler _mainThreadScheduler;
```

Comment:
Since these properties are `protected`, probably worth changing the name to UpperCamelCase to follow name conventions. This should be applied to all protected properties of this class.